bot.js

```diff
@@ -68,6 +67,9 @@ bot.on('message', function (message) {
                 break;
             }
             // Just add any case commands if you want to..
+            default: {
+                message.channel.stopTyping(true);
```

Comment:
lol, nice, just to be sure.
probably safe to remove shutup and stfu too

Reply:
im just gonna keep them there as a fallback.